### DISCUSSION TITLE
Publish to Windows store?

### DISCUSSION THREAD

**Discussion by JasonWei512** [CommunityToolkit/Labs-Windows#339] [Ideas]
UNO is great, but it's still different from running natively on Windows.
Maybe setting up a store publishing CI job for this?

**Comment by michael-hawker** [comment]
@JasonWei512 we're currently thinking we'll be blending this into the update of our main Windows Community Toolkit sample app (https://aka.ms/windowstoolkitapp) in the future when we release our next version. Though in the future we may think about having a dedicated app, it's not something we're thinking about at this moment.

But yes, we'd like to have it build in the CI and push to the store in the future in either case.

**Comment by kmgallahan** [comment]
@michael-hawker I agree that getting the sample app in the store would be great, and it would probably increase interest in the lab project.

The web experience is missing controls and can be janky compared to desktop, which gives an incomplete and inaccurate experience of the real deal.

Also, it just hangs in my main browser (Edge Dev, many extensions), so I have to open it elsewhere (Chrome, minimal extensions).

![Screenshot 2023-03-19 103803](https://user-images.githubusercontent.com/5379488/226183669-c0012549-e884-4495-8b79-e9101c7da9ce.jpg)
